### ISSUE TITLE
[2.5.2] Crash on clicking stats button

### ISSUE THREAD

**Issue by tajemniktv** [TheCSMods/mc-better-stats#45]
**Describe the bug**
Whenever you want to see statistics screen, the game crashes.

**To Reproduce**
1. Go to the statistics screen

**Expected behavior**
If applicable, a description of what you expected to happen.

**Logs**
https://mclo.gs/0zcN21c

**Comment by TheCSDev** [comment]
Hello and thanks for the bug report. Looking at the log file, the crash is likely caused by a conflict with:
`RingOfAttraction-Fabric-1.19.3-rc1-1.4.6`

And now that I look at that mod's file, it says 1.19.3, and  not 1.19.4. Maybe it could be that you are attempting to run that mod on 1.19.4 that is somehow conflicting with this mod and crashing the game?
Altho the CurseForge page for that mod does not explicitly state that mod can run on 1.19.4, the mod's code doesn't try to prevent it. But then again, I too don't bother with stopping the user from loading a mod of mine on another version, just in care a miracle happens and it works out..

In other words, I'd like to ask you to try and remove that mod and see if that fixes the crash issue. If it does, then we have a conflict issue that would need resolving.

**Comment by tajemniktv** [comment]
Hi! Did what you said, boss - Oddly that helped! I would never suspect that these two mods could be conflicting with each other... Thank you for all your work, these stats make my OCD happy

**Comment by TheCSDev** [comment]
I am glad that resolved your issue, and thanks for the feedback on this mod.

After further looking into the logs, I have come to the conclusion that the mod you removed was trying to interact with (Minecraft 1.19.3)-specific code while you had it installed on 1.19.4. In other words, I suppose your only option would be to wait for that mod to get a 1.19.4 port if you wish to use it. To be honest, I have no clue why the "broken" line of code in that mod triggers when opening the statistics screen.

Either way, I will now close this issue as resolved. Thanks again, and have a great day!
